packages/strip/src/index.js

```diff
@@ -44,15 +44,15 @@ export default function strip(options = {}) {
 
   const labels = options.labels || [];
 
-  const firstpass = new RegExp(`\\b(?:${functions.join('|')}|debugger)\\b`);
+  const firstpass = new RegExp(`\\b(?:${[...functions, ...labels].join('|')}|debugger)\\b`);
```

Comment:
I'd combine functions and labels into something that could be checked on line 55, e.g.:

```suggestion
  const targets = [...functions, ...labels];
  const firstpass = new RegExp(`\\b(?:${targets.join('|')}|debugger)\\b`);
```